packages/site/src/App.stories.tsx

```diff
@@ -9,12 +11,32 @@ import {
   getPaymentChannelsByLedgerMock,
 } from "./mocks/request";
 
+function createMockServer() {
+  const mockServer = new Server("ws://localhost:4005/api/subscribe");
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  mockServer.on("connection", () => {});
```

Comment:
Are you leaving this here in anticipation of mocking out some notification data in the future?

Reply:
This code has been removed in 977ad40. At first, I thought we needed a handler (even if the handler is empty). But that is not the case.